dependent/bundesgesundheitsblatt-gesundheitsforschung-gesundheitsschutz.csl

```diff
@@ -1,15 +1,14 @@
 <?xml version="1.0" encoding="utf-8"?>
 <style xmlns="http://purl.org/net/xbiblio/csl" version="1.0" default-locale="de-DE">
-  <!-- Generated with https://github.com/citation-style-language/utilities/tree/master/generate_dependent_styles/data/springer-fachzeitschriften-medizin -->
+  <!-- Generated with https://github.com/citation-style-language/utilities/tree/master/generate_dependent_styles/data/springer -->
   <info>
-    <title>Bundesgesundheitsblatt - Gesundheitsforschung - Gesundheitsschutz (German)</title>
-    <title-short>Bundesgesundheitsblatt Gesundheitsforschung Gesundheitsschutz</title-short>
+    <title>Bundesgesundheitsblatt - Gesundheitsforschung - Gesundheitsschutz</title>
```

Comment:
> I think it is also missing everywhere else. I guess we can delete the redundant short-titles but may want to keep the rest.

I think most of the cases where we lose the abbreviated title are from (former) BMC styles. If we do end up retiring the https://github.com/citation-style-language/utilities/tree/master/generate_dependent_styles/data/bmc folder, we can first retrieve the old abbreviations from https://github.com/citation-style-language/utilities/blob/master/generate_dependent_styles/data/bmc/_journals.tab.